lib/bundler/definition.rb

```diff
@@ -256,9 +256,30 @@ def lock(file)
         "#{File.expand_path(file)}"
     end
 
+    # Returns the version of Bundler that is creating or has created
+    # Gemfile.lock. Used in #to_lock.
+    def lock_version
+      lockfile = @lockfile_contents
+      lock_ver = nil
+      curr_ver = nil
+
+      lock_ver = @lockfile_contents[/^  \[(.*)\]$/, 1] if lockfile
```

Comment:
and I lied, this needs to be @lockfile_contents before those three lines can be removed.